bindsnet/network/topology.py

```diff
@@ -562,11 +571,13 @@ def __init__(self, source: Nodes, target: Nodes, nu: Optional[Union[float, Seque
         super().__init__(source, target, nu, weight_decay, **kwargs)
 
         self.w = kwargs.get('w', None)
-
         if self.w is None:
-            self.w = (torch.randn(1)[0] + 1) / 10
+            if self.wmin == -np.inf or self.wmax == np.inf:
```

Comment:
This also needs clamping/rescaling, to keep consistent with other connections?

Reply:
Yup.